src/utils/compressionUtils.ts

```diff
@@ -0,0 +1,9 @@
+export async function compress(dataString: string): Promise<Uint8Array> {
```

Comment:
Either is fine with me.

Actually we do use zip files for the `download` terminal command when downloading multiple files. So potentially we could reuse some utilities between the two (doesn't really matter though because the `download` terminal command doesn't really need to be optimized as it's very rare in usage)